manager/controllers/advisories.go

```diff
@@ -9,6 +9,8 @@ import (
 	"net/http"
 )
 
+var AdvisoriesSortFields = []string{"name", "description", "synopsis", "summary", "solution", "public_date"}
```

Comment:
Is the sort on text fields needed? We had issues with Katello in this area in the past, as the number of errata grows.

Reply:
As @mkoura mentioned before, we should also update openapi spec when adding filtering.